src/app/pages/RoflAppDetailsPage/index.tsx

```diff
@@ -200,12 +200,20 @@ export const RoflAppDetailsViewSearchResult: FC<{
 
   return (
     <StyledDescriptionList>
+      <dt>{t('common.name')}</dt>
+      <dd>
+        <RoflAppLink
+          id={app.id}
+          name={app.metadata['net.oasis.rofl.name']}
+          network={app.network}
+          withSourceIndicator={false}
+        />
+        <CopyToClipboard value={app.id} />
```

Comment:
I would copy the app name, because you may search for a substring and the app name may be way longer and you want to copy its name, e.g. to avoid typos.

Reply:
I think this will make https://github.com/oasisprotocol/explorer/issues/2332 more confusing